packages/api/src/models/soilAmendmentTaskProductsModel.js

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import Model from './baseFormatModel.js';
+import productModel from './productModel.js';
+import soilAmendmentTaskModel from './soilAmendmentTaskModel.js';
+
+class SoilAmendmentTaskProducts extends Model {
+  static get tableName() {
+    return 'soil_amendment_task_products';
+  }
+
+  static get idColumn() {
+    return 'id';
+  }
+
+  // Optional JSON schema. This is not the database schema! Nothing is generated
+  // based on this. This is only used for validation. Whenever a model instance
+  // is created it is checked against this schema. http://json-schema.org/.
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [],
+
+      properties: {
+        task_id: { type: 'integer' },
+        product_id: { type: 'integer', minimum: 0 },
```

Comment:
Oh, interesting. I copied all these fields verbatim from `soilAmendmentTaskModel`, and maybe quantity was originally intended, but since the other product quantities don't have, I just removed it altogether.